lib/reporters/cli/index.js

```diff
@@ -24,6 +25,19 @@ colors.setTheme({
     error: 'red'
 });
 
+extractSNR = function (executions) {
+    var snr = {};
+
+    _.isArray(executions) && executions.forEach(function (execution) {
```

Comment:
If the intent is to find the last existing definition of `nextRequest`, use `_.forEachRight` instead, with a bailout on the first hit.